docs/source/development/linting.md

```diff
@@ -72,36 +76,44 @@ These hooks are run before committing your code to your repositories to automati
 making code reviews easier and less time-consuming.
 
 ### Install `pre-commit`
+
 You can install `pre-commit` along with other dependencies by including it in the `requirements.txt` file of your
 Kedro project by adding the following line:
+
 ```text
 pre-commit
 ```
 You can also install `pre-commit` using the following command:
+
 ```bash
 pip install pre-commit
 ```
+
 ### Add `pre-commit` configuration file
+
 Create a file named `.pre-commit-config.yaml` in your Kedro project root directory. You can add entries for the hooks
 you want to run before each `commit`.
-Below is a sample `YAML` file with entries for `ruff` and black`:
+Below is a sample `YAML` file with entries for `ruff`:
+
 ```yaml
 repos:
   - repo: https://github.com/astral-sh/ruff-pre-commit
     # Ruff version.
-    rev: v0.0.270
+    rev: '' # Replace with latest version
     hooks:
       - id: ruff
-
-  - repo: https://github.com/psf/black
-    rev: 22.8.0
-    hooks:
-      - id: black
-        language_version: python3.9
+        args: [--fix]
+      - id: ruff-format
 ```
+
+The latest configuration for `ruff`'s pre-commit is available [here](https://github.com/astral-sh/ruff-pre-commit).
```

Comment:
See Vale message. We don't like to put links under the word "here" as it's vague and not very accessible for people using screen readers. Can I suggest you reword to "See GitHub for [the latest configuration for ruff's pre-commit]." and link as I've done?